pkg/cloudprovider/aws/instancetypes_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("InstanceTypes", func() {
 			ec2api := getInstanceTypeProviderMocks([]string{testZone}, []string{"m5.large"})
 			instanceTypeProvider := cloudprovideraws.NewInstanceTypeProvider(ec2api)
 			zonalSubnetOptions := map[string][]*ec2.Subnet{testZone: nil}
-			constraints := &cloudprovider.Constraints{}
+			constraints := &cloudprovideraws.AWSConstraints{Constraints: &cloudprovider.Constraints{}}
```

Comment:
Here's that stutter I was talking about :)

Reply:
ha so much